flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/TableEnvHiveConnectorTest.java

```diff
@@ -171,6 +172,57 @@ public void testDecimal() throws Exception {
 		}
 	}
 
+	@Test
+	public void testStaticPartition() throws Exception {
+		hiveShell.execute("create database db1");
+		try {
+			hiveShell.execute("create table db1.src (x int)");
+			hiveShell.insertInto("db1", "src").addRow(1).addRow(2).commit();
+			hiveShell.execute("create table db1.dest (x int) partitioned by (p1 string, p2 double)");
+			TableEnvironment tableEnv = getTableEnvWithHiveCatalog();
+			tableEnv.sqlUpdate("insert into db1.dest partition (p1='1''1', p2=1.1) select x from db1.src");
```

Comment:
I'm wondering what this means and why a user might do this: p1='1''1'

Reply:
Agree, a little confusing.

Reply:
I agree this may not be common use case, but theoretically a string partition column can take any valid values including those with single quotes. And Hive also supports this.